collection-scripts/gather

```diff
@@ -117,5 +117,8 @@ oc adm inspect --dest-dir must-gather --rotated-pod-logs "${all_ns_resources[@]}
 # Gather Performance profile information
 /usr/bin/gather_ppc
 
+# Gather Number of running VMs
+/usr/bin/gather_running_vms
```

Comment:
```suggestion
/usr/bin/gather_running_vms
```
this comment is redundant. The name of the function should tell the reader everything she needs to know about what the function is going to do. Let us avoid future comment rot.

Reply:
I don't mind removing it, I just tried the follow the pattern in this [file](https://github.com/openshift/must-gather/blob/197b7c6051deca66c76ecddc69c6b726d0d0f092/collection-scripts/gather#L67), which has comments above every execution of a script.

Reply:
I don't like many of those comments too, but I was not their reviewer...